app/Data/Wallet/WalletData.php

```diff
@@ -70,6 +74,11 @@ public static function fromModel(Wallet $wallet): self
             ],
             isRefreshingCollections: (bool) $wallet->is_refreshing_collections,
             canRefreshCollections: $wallet->canRefreshCollections(),
+            hasErc1155Nfts: Cache::remember(
+                'wallets:'.$wallet->id.':has-erc1155',
+                now()->addMinutes(30),
+                fn () => $wallet->allNfts()->where('type', TokenType::Erc1155)->exists()
```

Comment:
in case you add the scope this can be updated

Reply:
this checks for ERC1155, not ERC721 :trollface: but I'd leave out the scope for this one to be explicit